tracing-subscriber/src/filter/targets.rs

```diff
@@ -313,6 +313,36 @@ impl Targets {
             Interest::never()
         }
     }
+
+    /// Returns whether a [target]-[`Level`] pair would be enabled
+    /// for this `Targets`.
+    ///
+    /// You can use [`module_path!`] from `std` as the target
+    /// if you want to emulate the behavior of the
+    /// [`tracing::event!`] suite of macros.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use tracing_subscriber::filter::{Targets, LevelFilter};
+    /// use tracing_core::Level;
+    ///
+    /// let filter = Targets::new()
+    ///     .with_target("my_crate", Level::INFO)
+    ///     .with_target("my_crate::interesting_module", Level::DEBUG);
+    ///
+    /// assert!(filter.enabled_for("my_crate", &Level::INFO));
+    /// assert!(!filter.enabled_for("my_crate::interesting_module", &Level::TRACE));
+    /// ```
+    ///
+    /// [target]: tracing_core::Metadata::target
+    /// [`tracing::event!`]: tracing::event!
+    /// [`module_path!`]: std::module_path!
+    pub fn enabled_for(&self, target: &'static str, level: &Level) -> bool {
```

Comment:
nit, take it or leave it: it seems unnecessarily restrictive to take an `&'static str` here? actual span/event targets will only ever be static strings, but if a user is using this separately, they might want a non-`'static` string here?
```suggestion
    pub fn enabled_for(&self, target: &str, level: &Level) -> bool {
```

Reply:
Oh this is an oversight, it shouldn't be static, that was from when i was going to implement this with a metadata macro